app/collections/preview.cjsx

```diff
@@ -28,17 +28,18 @@ module.exports = React.createClass
       @setState {subjects}
 
   render: ->
-    [owner, name] = @props.collection.slug.split('/')
-
+    baseLink = ""
+    if @props.project?
```

Comment:
Ah good point - used by Subjects, and Zoo-wide Talk doesn't have Subjects. Will fix.